March 11th polls (#2587)
makerdao/community: governance/polls/Adjust SparkLend D3M Parameters - March 11, 2024.md

---
title: Adjust SparkLend D3M Parameters - March 11, 2024

> summary: Signal your support or opposition to increasing the SparkLend DAI Direct Deposit Module (D3M) debt ceiling to 2.5 billion DAI.

Reviewer: ```suggestion
summary: Signal your support or opposition to increasing the SparkLend DAI Direct Deposit Module (D3M) Maximum Debt Ceiling to 2.5 billion DAI.
```

discussion_link: https://forum.makerdao.com/t/mar-6-2024-proposed-changes-to-sparklend-for-upcoming-spell/23791
parameters:
    input_format: single-choice
    victory_conditions:
        - { type : plurality }
    result_display: single-vote-breakdown
version: v2.0.0
options:
   0: Abstain
   1: Yes
   2: No
start_date: 2024-03-11T16:00:00
end_date: 2024-03-14T16:00:00
---
# Poll: Adjust SparkLend D3M Parameters - March 11, 2024

The Governance Facilitators have placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of SparkLend. The community can vote in this poll to express support or opposition to using the SparkLend Direct Deposit DAI Module (D3M) with the listed parameters. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, March 11 at 16:00 UTC.

**This is a binary vote.**
- **You may vote for a single option.**
- **You should vote for the option which you prefer.**
- **If you would accept either option, you should vote 'Abstain'.**

## Review

> The community can vote in this poll to express support or opposition to making the following parameter changes to the $protocol_name DAI Direct Deposit Module (D3M):

Reviewer: ```suggestion
The community can vote in this poll to express support or opposition to making the following parameter changes to the SparkLend DAI Direct Deposit Module (D3M):
```

> * Increase the SparkLend D3M [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 1.25 billion DAI to **2.5 billion DAI**.

Reviewer: add difference and use definition from the MIP

```suggestion
* Increase the SparkLend D3M [Maximum Debt Ceiling](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-1-maximum-debt-ceiling-line-) by 1.25 billion DAI from 1.25 billion DAI to **2.5 billion DAI**.
```
